src/System.Net.Http.WinHttpHandler/src/System/Net/Http/WinHttpHandler.cs

```diff
@@ -1337,6 +1337,12 @@ private RendezvousAwaitable<int> InternalSendRequestAsync(WinHttpRequestState st
                     0,
                     state.ToIntPtr()))
                 {
+#if DEBUG
+                    int lastError = Marshal.GetLastWin32Error();
+                    Debug.Assert((unchecked((int)lastError) != Interop.WinHttp.ERROR_INSUFFICIENT_BUFFER &&
+                        unchecked((int)lastError) != unchecked((int)0x80090321)), // SEC_E_BUFFER_TOO_SMALL
```

Comment:
Should SEC_E_BUFFER_TOO_SMALL just be added to Interop.WinHTTP as a named const?